.github/workflows/release.yaml

```diff
@@ -6,27 +6,12 @@ on:
       - main
 
 jobs:
-  lib-check:
-    name: Check libraries
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: Check libs
-        uses: canonical/charming-actions/check-libraries@2.1.1
-        with:
-          credentials: "${{ secrets.CHARMHUB_TOKEN }}" # FIXME: current token will expire in 2023-07-04
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
-
```

Comment:
Why not keep this? It is not an expensive operation and there may be a few days between the check libs ran on the feature branch and the merging to main.

Reply:
Because it will be executed twice.
It is still running as a part of:
```
ci-tests:
    uses: ./.github/workflows/ci.yaml"
```